chart/templates/proxy_injector.yaml

```diff
@@ -52,11 +52,14 @@ spec:
             path: /ready
             port: 9995
           failureThreshold: 7
-        {{- if .EnableHA }}
+        {{ with .ProxyInjectorResources -}}
         resources:
           requests:
-            cpu: 20m
-            memory: 50Mi
+            {{- with .CPU.Request}}cpu: {{.}}{{end}}
```

Comment:
Curious as to why we aren't using the `resource` template here?

Reply:
oversight, good catch